### PR TITLE
[DOC] Update to standalone operator deployment sections

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

Updates the _Alternative standalone deployment options for Strimzi Operators_ section in the _Deploying Strimzi_ guide
The standalone deployment procedures now show example configuration.
Section provides more context on when standalone deployment is useful

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Comment by PaulRMellor** [comment]
@scholzj @tombentley I struggled to make sense of the TLS config from the original text. I've tried to make clearer, but I think it still needs work.

File: documentation/modules/deploying/proc-deploy-user-operator-standalone.adoc [hunk 1]
```diff
@@ -5,59 +5,130 @@
...
+This must be the same ZooKeeper cluster that your Kafka cluster is using.
+<9> The ZooKeeper session timeout, in milliseconds.
+The default is `20000` (18 seconds).
+<10> The level for printing logging messages.
+You can set the level to `ERROR`, `WARNING`, `INFO`, `DEBUG`, or `TRACE`.
+<11> Enables garbage collection (GC) logging.
+The default is `true`.
+<12> The validity period for the Certificate Authority.
+The default is `365` days.
+<13> The renewal period for the Certificate Authority. The renewal period is measured backwards from the expiry date of the current certificate.
```
Comment (PaulRMellor) [inline documentation/modules/deploying/proc-deploy-user-operator-standalone.adoc:98]: Manual renewal?

**Review by tombentley — APPROVED** [review]
No further comments beyond the ones Jakub has already made.

File: documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc [hunk 2]
```diff
@@ -6,10 +6,17 @@
 [id="deploy-create-cluster_{context}"]
 = Create the Kafka cluster
 
-In order to create your Kafka cluster, you deploy the Cluster Operator to manage the Kafka cluster, then deploy the Kafka cluster.
+[role="_abstract"]
+To manage a Kafka cluster with the Cluster Operator, you deploy it as a `Kafka` resource.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc:10]: I changed to: _To be able to manage a Kafka cluster with the Cluster Operator, you must deploy it as a `Kafka` resource._

File: documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc [hunk 3]
```diff
@@ -6,10 +6,17 @@
...
+Strimzi provides example deployment files to do this.
+You can use these files to deploy the Topic Operator and User Operator at the same time.
 
-When deploying the Kafka cluster using the `Kafka` resource, you can deploy the Topic Operator and User Operator at the same time.
-Alternatively, if you are using a non-Strimzi Kafka cluster, you can deploy the Topic Operator and User Operator as standalone components.
+Alternatively, you can deploy the Topic Operator and User Operator as standalone components.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc:14]: I've rewritten this section:

> If you haven't deployed a Kafka cluster as a `Kafka` resource, you can't use the Cluster Operator to manage it. This applies, for example, to a Kafka cluster running outside of Kubernetes.  
> But you can deploy and use the Topic Operator and User Operator as standalone components.

File: documentation/assemblies/deploying/assembly-deploy-standalone-operators.adoc [hunk 4]
```diff
@@ -5,10 +5,14 @@
 [id="deploy-standalone-operators_{context}"]
 = Alternative standalone deployment options for Strimzi Operators
 
-When deploying a Kafka cluster using the Cluster Operator, you can also deploy the Topic Operator and User Operator.
-Alternatively, you can perform a standalone deployment.
+[role="_abstract"]
+As an alternative to deploying the Topic Operator and User Operator with the Cluster Operator, you can perform a standalone deployment.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/assemblies/deploying/assembly-deploy-standalone-operators.adoc:9]: Reduced it to:

> You can perform a standalone deployment of the Topic Operator and User Operator.

File: documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc [hunk 5]
```diff
@@ -5,59 +5,120 @@
...
 = Deploying the standalone Topic Operator
 
-This procedure shows how to deploy the Topic Operator as a standalone component.
+[role="_abstract"]
+This procedure shows how to deploy the Topic Operator as a standalone component to manage topics.
+You can use a standalone Topic Operator with a Kafka cluster that is not managed by the Cluster Operator.
 
-A standalone deployment requires configuration of environment variables, and is more complicated than xref:deploying-the-topic-operator-using-the-cluster-operator-str[deploying the Topic Operator using the Cluster Operator].
-However, a standalone deployment is more flexible as the Topic Operator can operate with _any_ Kafka cluster, not necessarily one deployed by the Cluster Operator.
+A standalone deployment is more complicated than xref:deploying-the-topic-operator-using-the-cluster-operator-str[deploying the Topic Operator using the Cluster Operator].
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc:12]: True. I've reduced to this in both procedures:

> A standalone deployment can operate with any Kafka cluster.

File: documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc [hunk 6]
```diff
@@ -5,59 +5,120 @@
...
+However, a standalone deployment is more flexible as the Topic Operator can operate with any Kafka cluster, not necessarily one deployed by the Cluster Operator.
+
+A standalone deployment file is provided.
+Configure the deployment file to add the environment variables that enable the Topic Operator to connect to a Kafka cluster.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc:16]: Edit's fine with me. I've specified the file as we change the sentence before it to _Standalone deployment files are provided._

> Edit the `05-Deployment-strimzi-user-operator.yaml` deployment file ...

File: documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc [hunk 7]
```diff
@@ -5,59 +5,120 @@
...
+            - name: STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS <5>
+              value: "20000"
+            - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS <6>
+              value: "900000"
+            - name: STRIMZI_TOPIC_METADATA_MAX_ATTEMPTS <7>
+              value: "6"
+            - name: STRIMZI_LOG_LEVEL <8>
+              value: INFO
+            - name: STRIMZI_TLS_ENABLED <9>
+              value: "true"
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc:64]: Changed

File: documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc [hunk 8]
```diff
@@ -5,59 +5,120 @@
...
+      containers:
+        - name: strimzi-topic-operator
+          # ...
+          env:
+            - name: STRIMZI_NAMESPACE <1>
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: STRIMZI_KAFKA_BOOTSTRAP_SERVERS <2>
+              value: my-cluster-kafka-bootstrap:9092
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc:50]: Good idea. Updated.

**Review by scholzj — APPROVED** [review]

File: documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc [hunk 9]
```diff
@@ -5,59 +5,119 @@
 [id='deploying-the-topic-operator-standalone-{context}']
 = Deploying the standalone Topic Operator
 
-This procedure shows how to deploy the Topic Operator as a standalone component.
+[role="_abstract"]
+This procedure shows how to deploy the Topic Operator as a standalone component to manage topics.
```
Comment (laidan6000) [inline documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc:9]: _Suggested change:_

to manage topics > for topic management

**Review by laidan6000 — APPROVED** [review]
Looks great.

File: documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc (hunk 9 above)
Comment (PaulRMellor) [reply documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc:9]: Changed the standalone user procedure intro to the same

**Comment by PaulRMellor** [comment]
@strimzi/maintainers please cherry-pick to 0.24 branch

**Pull request merged**